Client/Frontend/Browser/SimpleToast.swift

```diff
@@ -19,8 +19,23 @@ struct SimpleToastUX {
     static let BottomToolbarHeight = CGFloat(45)
 }
 
-struct SimpleToast {
-    func showAlertWithText(_ text: String, image: String, bottomContainer: UIView) {
+final class SimpleToast {
```

Comment:
Not so simple anymore, hum 😉 ? But I guess that's how it goes when we add features to it 😆